.readthedocs.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+
+build:
+  os: ubuntu-22.04
```

Comment:
Confirm on their website, but I think this is the latest container.
```suggestion
  os: ubuntu-24.04
```

Reply:
Yes, that is!